docs/docs/03-Unity Tutorial/03-part-2.md

```diff
@@ -23,7 +23,7 @@ In your `blackholio` directory, run the following command to initialize the Spac
 Run the following command to initialize the SpacetimeDB server module project with Rust as the language:
 
 ```bash
-spacetime init --lang=rust server-rust
+spacetime init --lang rust server-rust
```

Comment:
In this case I'm being prompted for a project path, I think most users would be smart enough to just press enter to do the default thing but maybe it would be good to just include the `--project-path` option here so that users don't get prompted?